md/terminusx/start-with-a-client.md

```diff
@@ -58,12 +58,12 @@ client.connect().then(()=>{
 ```python
 
 from terminusdb_client import WOQLClient
-user     = f"jimbo"
-team     = f"logicistics" # My team name.
-endpoint = f"https://cloud-dev.dcm.ist/{team}/"
+user     = "jimbo"
+team     = "logicistics" # My team name.
+endpoint = "https://cloud-dev.dcm.ist/{team}/"

```

Comment:
This one needs to be a f-string right?

Reply:
yes, only that one and I don't know why I remove it

Reply:
nice spot